src/components/LoadingBar/LoadingBar.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'react-emotion';
+
+import { childrenPropType } from '../../util/shared-prop-types';
+import { textKilo } from '../../styles/style-helpers';
+import id from '../../util/unique-id';
+
+const calculateSize = ({ theme, size }) => {
+  const sizeMap = {
+    kilo: theme.spacings.byte,
+    mega: theme.spacings.mega,
+    giga: theme.spacings.tera
+  };
+  return sizeMap[size];
+};
+
+const baseStyles = ({ theme }) => css`
+  label: loading-bar;
+  display: flex;
+  align-items: center;
+  margin-bottom: ${theme.spacings.mega};
+`;
+
+const progressStyles = ({ theme, size, value, max }) => css`
```

Comment:
Let's store the border radius in a variable so you can dynamically calculate the inner radius.